vantage/manifest.json

```
@@ -0,0 +1,46 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "f9d211f9-8cd2-4bab-9718-462c5a3fa9ad",
+  "app_id": "vantage",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "This integration allows Datadog users to import their Datadog costs into Vantage.",
+    "title": "Vantage",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Datadog Monthly Costs with Budget",
+        "image_url": "assets/images/vantage-datadog-budget-forecast.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Datadog Costs - Grouped by Service",
+        "image_url": "assets/images/vantage-datadog-grouped-report.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Provider Comparison Overview",
+        "image_url": "assets/images/vantage-datadog-provider-summary.png"
+      }
+    ],
+    "classifier_tags": [
+      "Supported OS::Linux",
+      "Supported OS::Mac OS",
+      "Supported OS::Windows",
+      "Offering::Integration"
```

```suggestion
      "Offering::Integration",
      "Category::Cost Management",
      "Category::Cloud"
      
```
Suggesting a few categories to help with discoverability